static/src/js/components/GranuleResults/GranuleResultsFocusedMeta.js

```diff
@@ -0,0 +1,407 @@
+                        <Button
+                          className="granule-results-focused-meta__image-nav-button"
+                          type="button"
+                          icon={FaChevronLeft}
+                          label="Previous thumbnail"
+                          onClick={() => onClickPreviousButton()}
+                        />
+                        <Button
+                          className="granule-results-focused-meta__image-nav-button"
+                          type="button"
+                          icon={FaChevronRight}
+                          label="Next thumbnail"
+                          onClick={() => onClickNextButton()}
+                        />
+                      </div>
+                      <div className="granule-results-focused-meta__image-nav-secondary">
+                        <span className="granule-results-focused-meta__pagination">
+                          {`${activeBrowseImageIndex + 1}/${browseThumbnails.length}`}
+                        </span>
+                        {browseImageSelectionPopover}
+                      </div>
+                    </nav>
+                  )
+                }
+                <div className="granule-results-focused-meta__thumb-container">
+                  {
+                    focusedGranuleId && browseThumbnails.map(({
+                      href,
+                      description = ''
+                    }, i) => {
+                      const thumbnailClassName = classNames([
+                        'granule-results-focused-meta__thumb',
+                        {
+                          'granule-results-focused-meta__thumb--is-active': activeBrowseImageIndex === i
+                        }
+                      ])
+
+                      // Preload each image as they are first rendered so they are more likely to be
+                      // done loading prior to the user navigating to the next image
+                      let preloadImg = new Image()
+                      preloadImg.src = href
+                      preloadImg.addEventListener('load', () => { preloadImg = undefined })
+
+                      const imgSrc1x = `${getEarthdataConfig(earthdataEnvironment).cmrHost}/browse-scaler/browse_images/granules/${focusedGranuleId}?h=150&w=150&imageSrc=${href}`
+                      const imgSrc2x = `${getEarthdataConfig(earthdataEnvironment).cmrHost}/browse-scaler/browse_images/granules/${focusedGranuleId}?h=300&w=300&imageSrc=${href}`
+
+                      return (
+                        href && (
+                          <EDSCImage
+                            key={href}
+                            className={thumbnailClassName}
+                            srcSet={`${imgSrc1x} 1x, ${imgSrc2x} 2x`}
+                            src={imgSrc1x}
+                            alt={description || `Browse image for ${title}`}
+                            width="150px"
```

Comment:
For the case where it is `imgSrc2x` does this width& height need to also be set to 300?

Reply:
Nope, it will still be displayed at 150x150